### ISSUE TITLE
Bug - Modal - aria-hidden is not removed after closing modal-based form

### ISSUE THREAD

**Issue by agagancarczyk** [patternfly/patternfly-react#8925]
**Describe the problem**
The bug in the Patternfly component used in the Keycloak UI is causing [serious a11y violation](https://github.com/keycloak/keycloak/issues/19319). 

I tried to [fix it in our Keycloak UI codebase](https://github.com/keycloak/keycloak/pull/19472/files) but it's something that needs to be fixed in the PattternFly library.

**How do you reproduce the problem?**
This can be reproduced in Keycloak UI as per the steps outlined in [this issue](https://github.com/keycloak/keycloak/issues/19319).

**Expected behavior**
This a11y serious violation is not detected by the Deque Axe Browser tool.

**Is this issue blocking you?**
Yes, it's blocking us as it's the last serious a11y violation we need to address.

**Screenshots**
All screenshots are in the issues mentioned above.

**What is your environment?**
 - OS: [Mac iOS]
 - Browser [chrome]
 - Version [latest]

**What is your product and what release date are you targeting?** Keycloak UI

**Any other information?**
